hoot-cmd/src/main/cpp/hoot/cmd/HelpCmd.cpp

```diff
@@ -5,7 +5,7 @@
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
- *
+ * 
```

Comment:
I've noticed that this extra space that has been added comes and goes with different commits.  It appears as though the license scripts add this into the header while some developers' commits remove them.  Looking in the repository there are quite a few (20% - 30%) files that don't have this space and the remaining (70% - 80%) that do have this space.  We need to standardize.  If you look at the other lines within the copyright header you'll see that this is the only line that contains ` * ` instead of ` *`.  Another thing to take into consideration is that by default QtCreator removes trailing spaces on any line that you modify.  I vote that we remove them to make them all consistent with all the other lines in the copyright header.